utils/oominfo/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+  "log"
+  "github.com/google/cadvisor/utils/oominfo/oomparser"
```

Comment:
nit: empty line before this one. We separate the Go standard imports from the non-standard ones with an empty line.